src/chrome/content/rules/Notepad++.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+
+	Non-functional subdomain:
+		- docs
+
+-->
+
+<ruleset name="Notepad++">
+
+	<target host=    "notepad-plus-plus.org" />
+	<target host="www.notepad-plus-plus.org" />
+
+  	<securecookie host="^notepad-plus-plus\.org$" name=".*" />
+
+	<!-- www doesn't serve HTTPS -->
+	<rule from="^http://www\.notepad-plus-plus\.org/"
+		to="https://notepad-plus-plus.org/" />
+
+		<test url="http://notepad-plus-plus.org/" />
```

Comment:
I see. I add the test url out of habit. Now I can better understand purpose of test url.